Sources/SwiftDocCUtilities/TemplateLibrary/Init/___FILEBASENAME_INIT___.docc/Documentation.md

```diff
@@ -0,0 +1,33 @@
+# Documentation
```

Comment:
Should this be `___DOCUMENTATION_NAME___` or do we just always use "Documentation" as the title of the catalog?

Reply:
I was using `Documentation` as the default title but because of your comment I decided to change it for `___DOCUMENTATION_TITLE___ `so it is  clearer that it works as a placeholder in the template.

The user can still change the title using the `--documentation-title` option.

Resolved in https://github.com/apple/swift-docc/pull/719/commits/2a993c4e59d8222a3359af61a8b4ef0b3be9bdcf